### PR TITLE
[Reference cleanup]LPS-162575

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to convert method references to field references in `journal-service`. Thanks!

https://issues.liferay.com/browse/LPS-162575

### CODE REVIEW

**Comment by JanisZhang** [comment]
Hi @Kvale1733, As Catarine is on PTO, Could you please help to scan the test results? Thank you

**Comment by Kvale1733** [comment]
@JanisZhang No regressions found

File: modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/upgrade/registry/JournalServiceUpgradeStepRegistrator.java [hunk 1]
```diff
@@ -339,14 +339,6 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 				_groupLocalService));
 	}
 
-	@Reference(unbind = "-")
-	protected void setPortalCapabilityLocator(
-		PortalCapabilityLocator portalCapabilityLocator) {
-
-		// See LPS-82746
```
Comment (dantewang) [inline modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/upgrade/registry/JournalServiceUpgradeStepRegistrator.java:346]: @JanisZhang Could you keep this comment? Does SF forbid it?

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by JanisZhang** [comment]
Hi @dantewang, I only add a comment in [/JournalServiceUpgradeStepRegistrator.java](https://github.com/liferay-core-infra/liferay-portal/pull/1620/commits/367b1531ac5fa347a25b2d161ecfe7ad5f4e9461#diff-47bc0fd8f1195aaa3aab915e6dd5a7daf4f7ee1ab45c360ae35658d3b689d315), no need to redo `ci:test`. Thanks

**Review by dantewang — APPROVED** [review]
@shuyangzhou @tinatian Forwarding this one as all tests passed and changes are direct replacements.

**Comment by dantewang** [comment]
ci:forward

**Comment by catarineaguiar** [comment]
no regressions found.

**Pull request closed without merging**